source/features/clean-issue-filters.tsx

```diff
@@ -22,16 +17,11 @@ const getCounts = cache.function(async (): Promise<Counts> => {
 		allowErrors: true
 	});
 
-	return {
-		projects:
-			(repository.projects.totalCount as number) +
-			(organization?.projects?.totalCount as number ?? 0),
-		milestones: repository.milestones.totalCount
-	};
+	return ((repository.projects.totalCount as number) + (organization?.projects?.totalCount as number ?? 0)) > 0;
```

Comment:
Why add numbers when we return a boolean? :)

I think: 
```suggestion
	return Boolean(repository.projects.totalCount) && Boolean(organization?.projects?.totalCount);
```